CIP-0072/README.md

```diff
@@ -0,0 +1,412 @@
+                    "version": {
+                      "type": "integer"
+                    },
+                    "plutusVersion": {
+                      "type": "integer"
+                    },
+                    "fullScriptHash": {
+                      "type": "string"
+                    },
+                    "scriptHash": {
+                      "type": "string"
+                    },
+                    "contractAddress": {
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "version",
+                    "plutusVersion",
+                    "fullScriptHash",
+                    "scriptHash",
+                    "contractAddress"
+                  ]
+                }
+              ]
+            },
+            "audits": {
+              "type": "array",
+              "items": [
+                {
+                  "type": "object",
+                  "properties": {
+                    "auditId": {
+                      "type": "string"
+                    },
+                    "auditor": {
+                      "type": "string"
+                    },
+                    "auditLink": {
+                      "type": "string"
+                    },
+                    "auditType": {
+                      "type": "string"
+                    },
+                    "signature": {
+                      "type": "string"
+                    },
+                    "publicKey": {
+                      "type": "string"
+                    }
+                  },
+                  "required": [
+                    "auditId",
+                    "auditor",
+                    "auditLink",
+                    "auditType",
+                    "signature",
+                    "publicKey"
+                  ]
+                }
+              ]
+            }
+          },
+          "required": [
+            "id",
+            "name",
+            "purpose",
+            "type",
+            "versions",
+            "audits"
+          ]
+        }
+      ]
+    }
+  },
+  "required": [
+    "subject",
+    "projectName",
+    "link",
+    "twitter",
+    "category",
+    "subCategory",
+    "description",
+    "releases",
+    "scripts"
+  ]
+}
+```
+
+## Example
+
+```json
+{
+    "subject": "9SYAJPNN",
+    "projectName": "My Project",
+    "link": "https://myProject.app",
+    "twitter": "https://twitter.com/MyProject",
+    "category": "GAMING",
+    "subCategory": "RPG",
+    "description": {
+                   "short": "A story rich game where choices matter"
+    },	
+
+    "releases": [
+        {
+            "releaseNumber": 1,
+            "releaseName": "V1",
+            "auditId": "z5L90f",
+            "scripts": [
+                {
+                    "id": "PmNd6w",
+                    "version": 1
+                }
+            ]
+        }
+    ],
+    "scripts": [
```

Comment:
@matiwinnetou As we discuss last i'm including a similar structure of the suggested metadata structure for review and more collaboration.